src/main/java/io/github/thebusybiscuit/slimefun4/implementation/resources/OrganicNetherGoo.java

```diff
@@ -0,0 +1,36 @@
+package io.github.thebusybiscuit.slimefun4.implementation.resources;
+import org.bukkit.inventory.ItemStack;
+
+import io.github.thebusybiscuit.slimefun4.api.items.ItemSetting;
+import io.github.thebusybiscuit.slimefun4.core.attributes.PiglinBarterDrop;
+import me.mrCookieSlime.Slimefun.Lists.RecipeType;
+import me.mrCookieSlime.Slimefun.Objects.Category;
+import me.mrCookieSlime.Slimefun.Objects.SlimefunItem.SlimefunItem;
+import me.mrCookieSlime.Slimefun.api.SlimefunItemStack;
+
+/**
+ * This {@link SlimefunItem} can only be obtained via bartering with a {@link Piglin}. 
+ *
+ * @author dNiym
+ *
+ * @see PiglinBarterDrop
+ *
+ */
+
+public class OrganicNetherGoo extends SlimefunItem implements PiglinBarterDrop {
+
+    private final ItemSetting<Integer> chance = new ItemSetting<>("barter-chance", 3);
```

Comment:
Should probably also add a "validateInput" method to make sure it's between 1-99.

Reply:
There is a line that checks this in the piglin listener.. do you think it should be done on a per item basis? Or on maybe the barter drop interface instead?
https://github.com/TheBusyBiscuit/Slimefun4/pull/2220/files/b27059240b6305e89bb058fdce15b62ba4d08019#diff-030aa5ca3b19598a3acad3d2ccc70609

Reply:
know that it is checked there.
But you should validate inputs like that on the ItemSetting level too, so that server owners will be notified that their settings are misconfigured.

In general, illegal values should be checked on all levels of implementation.